contents/docs/error-tracking/installation/flutter.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Flutter error tracking installation
+showStepsToc: true
+tableOfContents: [
+  {
+    url: 'install-posthog-flutter-sdk',
+    value: 'Install PostHog Flutter SDK',
+    depth: 1,
+  },
+  {
+    url: 'manually-capture-exceptions',
+    value: 'Manually capture exceptions',
+    depth: 1,
+  },
+  {
+    url: 'verify-error-tracking',
+    value: 'Verify error tracking',
+    depth: 1,
+  }
+]
+---
+
+import { Steps, Step } from 'components/Docs/Steps'
+import { CalloutBox } from 'components/Docs/CalloutBox'
+
+import InstallFlutter from "../../integrate/_snippets/install-flutter.mdx"
+import StepVerifyErrorTracking from "./_snippets/step-verify-error-tracking.mdx"
+
+<CalloutBox icon="IconInfo" title="Planned features" type="fyi">
+
+We currently don't support the following features:
```

Comment:
does it work on flutter web? is the stack trace human readable? otherwise this should be part of the list